script.js

```diff
@@ -0,0 +1,43 @@
+const buttonAdd = document.querySelector('#add');
+const bookTitle = document.querySelector('#title');
+const bookAuthor = document.querySelector('#author');
+const bookDisplay = document.querySelector('#display');
+const bookData = {};
+let bookIndex = 0;
+// remove book
+const removeEvent = (button, index) => {
+  button.addEventListener('click', function () {
+    delete bookData[index];
+    this.parentNode.remove();
+    window.localStorage.setItem('bookData', JSON.stringify(bookData));
+  });
+};
+
+// add book
+buttonAdd.addEventListener('click', () => {
+  const bookTitleValue = bookTitle.value;
+  const bookAuthorValue = bookAuthor.value;
+  bookData[bookIndex] = { title: bookTitleValue, author: bookAuthorValue };
+  bookIndex += 1;
+  const newDiv = document.createElement('div');
+  newDiv.innerHTML = `${bookData[bookIndex - 1].title}<br>${bookData[bookIndex - 1].author}<br> <button id='remove'>remove</button> <br> <br> <hr>  `;
+  bookDisplay.appendChild(newDiv);
+  removeEvent(newDiv.querySelector('#remove'), bookIndex - 1);
+  window.localStorage.setItem('bookData', JSON.stringify(bookData));
+});
```

Comment:
- [ ] [Optional] After adding a book to the collection, I would suggest clearing the input fields. It will improve the user experience of the app.